lib/InteractiveViewer.ml

```diff
@@ -83,11 +92,15 @@ let view (patches : string Patch.t list) =
                  | `ASCII 't', [] ->
                      toggle_view_mode ();
                      `Handled
+                 | `ASCII 'r', [] ->
+                     toggle_rendering_mode ();
+                     `Handled
                  | _ -> `Unhandled)
                (W.string
                   "Type 'h' to go to the help panel, 'q' to quit, 'n' to go to \
                    the next operation, 'p' to go to the previous operation. \
-                   Press 't' to toggle view mode.");
+                   Press 't' to toggle view mode. Press 'r' to toggle \
+                   rendering mode.");
```

Comment:
For a later PR! The help string is now wider than any reasonable terminal and is no longer readable. I think it should be rewritten in a more concise way, eg. `[h]elp [q]uit ...` or `[n/p] to navigate` or other variations. Also, it doesn't need to be exhaustive.